packages/components/src/standalone-comparison-grid/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { Button } from '@wordpress/components';
+import './style.scss';
+
+interface StandAloneComparisonGridColumn {
+	title: string;
+	rows: string[];
```

Comment:
```suggestion
	rows: ReactNode[];
```
Usually is better to receive ReactNodes instead of strings because it makes the component more flexible to receive formatted items like `rows: <strong>Feature X</strong>` 

It also enables us to introduce custom click events if necessary at no cost.

Reply:
That's a good idea!